ansible/roles/ironic/tasks/config.yml

```diff
@@ -77,8 +77,8 @@
   when:
     # Only required when Ironic inspector is in use.
     - groups['ironic-inspector'] | length > 0
-    - inventory_hostname in groups['ironic-pxe']
-    - not enable_ironic_ipxe | bool
+    - inventory_hostname in groups['ironic-pxe'] or
+        inventory_hostname in groups['ironic-ipxe']
```

Comment:
Removing the flag is correct, but I don't think we need to do this on ipxe hosts - only pxe.